src/node.js

```diff
@@ -388,6 +396,53 @@
       nextTickQueue.push(obj);
       tickInfo[kLength]++;
     }
+
+    // Internal callbacks that must be run when there is no more micro-tasks
+    // scheduled for this turn.
```

Comment:
@vkurchatkin the whole point of doing this here is running something after the microtask queue is finished because that's when @domenic believes it is reasonable to report unhandled rejections (for example - attaching it in a `.then` of a resolved promise should not report the false positive). If you have another suggestion of when to run this I'd love to hear it.

I didn't understand the extra closure allocation bit - isn't it just allocating one extra closure per full turn?